c_glib/arrow-glib/basic-data-type.cpp

```diff
@@ -2282,6 +2307,19 @@ garrow_fixed_shape_tensor_data_type_init(GArrowFixedShapeTensorDataType *object)
 static void
 garrow_fixed_shape_tensor_data_type_class_init(GArrowFixedShapeTensorDataTypeClass *klass)
 {
+  GParamSpec *spec;
+
+  auto gobject_class = G_OBJECT_CLASS(klass);
+  gobject_class->get_property = garrow_fixed_shape_tensor_data_type_get_property;
+
+  spec = g_param_spec_uint64("n_dimensions",
```

Comment:
Could you add a docstring for public properties?

```suggestion
  /**
   * GArrowFixedShapeTensorDataType::n-dimensions:
   *
   * The number of dimensions of tensor elements.
   *
   * Since: 21.0.0
   */
  spec = g_param_spec_uint64("n_dimensions",
```